test_cli.py

```diff
@@ -0,0 +1,46 @@
+"""
+Tests that arguments are parsed correctly by bin/kalite.
+Just try a few management commands and get their output.
+
+To run these tests, just run this script: `python test_cli.py`.
+It will return a nonzero value if the tests fail.
+"""
+import os
+
+from subprocess import check_output, check_call, CalledProcessError
+
+filepath = os.path.dirname(__file__)
+TEST_BASE = "python " + os.path.join(filepath, "bin", "kalite") + " manage test --test-cli "
+
+print TEST_BASE
```

Comment:
Yes. Consider this PR as something to stimulate the imagination, more than something which I wish to be included in the code base.